evdev/eventio.py

```diff
@@ -46,6 +46,60 @@ def read_loop(self):
             for event in self.read():
                 yield event
 
+    def read_loop_blockless(self, timeout=0.001):
+        '''
+        Enter a blockless :func:`select.select()` loop that yields input events endlessly.
+        '''
+        
+        # previous -> InputEvent.sec, InputEvent.usec, InputEvent.type, InputEvent.code, InputEvent.val
+        previous = InputEvent(sec=0, usec=0, type=0, code=0, value=0)
```

Comment:
your other variable is called `current_event`. For consistency this should be `previous_event`

Reply:
You mean at the `if current_event == None:` and `if current_event.type == syn_event:` clauses?

Reply:
I just mean the naming of the `previous` variable.

```py
previous_event = InputEvent(sec=0, usec=0, type=0, code=0, value=0)
```

Reply:
I see. I'm committing those changes soon. Again, thanks for the insights.